package.json

```diff
@@ -39,6 +42,7 @@
   "scripts": {
     "build": "gatsby build",
     "dev": "gatsby develop",
+    "serve": "gatsby serve",
```

Comment:
Why do we need this?

Reply:
Serve is mainly used when testing production builds, in my case I used it to test a build